simconvoi.h

```diff
@@ -495,7 +495,23 @@ class convoi_t : public sync_steppable, public overtaker_t
 	/// Pushes the convoy stopping time at the current halt to the schedule
 	void push_convoy_stopping_time();
 
+
+
+
+
 public:
+	// Reverses the order of the convoy.
+	// @author: jamespetts
+	bool reversing;// Whether this convoy's vehicles are currently arranged in reverse order. The flag for calculation of position.
```

Comment:
TODO: check the responsibility again.
```suggestion
	// Reverses the order of the convoy.
	// @author: jamespetts
	bool reversed_at_current_halt;
```